Core/include/Acts/Propagator/detail/SteppingHelper.hpp

```diff
@@ -51,9 +53,8 @@ Acts::Intersection3D::Status updateSingleSurfaceStatus(
     return Intersection3D::Status::onSurface;
   }
 
-  // Path and overstep limit checking
-  const double nearLimit = stepper.overstepLimit(state);
-  const double farLimit = state.stepSize.value(ConstrainedStep::aborter);
+  const double nearLimit = std::numeric_limits<double>::lowest();
+  const double farLimit = std::numeric_limits<double>::max();
```

Comment:
right since I removed the overstepping tolerance and I don't think using the aborter limit makes sense we can just use the intersection without any constraints